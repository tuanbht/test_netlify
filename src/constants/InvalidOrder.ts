export const ORDER_CANCELLED_CONTENT = {
  title: 'Order cancelled',
  subtitle: 'This order has been cancelled and is no longer available. To place a new order, contact your Merchant.',
};

export const ORDER_EXPIRED_CONTENT = {
  title: 'Order expired',
  subtitle: 'This order expired and is no longer available. To place a new order, contact your Merchant.',
};
