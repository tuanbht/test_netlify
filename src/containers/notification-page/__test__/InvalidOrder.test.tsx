import { shallow } from 'enzyme';
import React from 'react';
import InvalidOrder from '../index';

describe('InvalidOrder', () => {
  it('renders template correctly', () => {
    expect(
      shallow(<InvalidOrder content={{ title: 'invalid-order-title', subtitle: 'invalid-order-subtitle' }} />),
    ).toMatchSnapshot();
  });
});
