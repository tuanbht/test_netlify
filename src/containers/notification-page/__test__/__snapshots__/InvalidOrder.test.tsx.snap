// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InvalidOrder renders template correctly 1`] = `
<div
  className="makeStyles-root-1"
  id="mobile-screen"
>
  <Header
    showBack={false}
  />
  <div
    className="makeStyles-content-2"
  >
    <div
      className="makeStyles-title-3"
    >
      invalid-order-title
    </div>
    <div
      className="makeStyles-subtitle-4"
    >
      invalid-order-subtitle
    </div>
  </div>
</div>
`;
